tools/fuzzer/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+add_executable(sof-fuzzer
+	main.c
+	qemu-bridge.c
+	../testbench/topology.c
+	platform/byt-host.c
+)
+
+target_link_libraries(sof-fuzzer PRIVATE
+	"-lglib-2.0 -lrt"
```

Comment:
You have `target_link_libraries(sof-fuzzer INTERFACE ${GLIB_LDFLAGS})` so `-lglib-2.0` is not needed

Reply:
@jajanusz without the -lglib-2.0, I run into compilation errors as below. Could you please let me know what I may be missing?

CMakeFiles/sof-fuzzer.dir/qemu-bridge.c.o: In function `mq_init':
qemu-bridge.c:(.text+0x49e): undefined reference to `g_thread_new'
qemu-bridge.c:(.text+0x62e): undefined reference to `g_thread_new'
collect2: error: ld returned 1 exit status
fuzzer/CMakeFiles/sof-fuzzer.dir/build.make:172: recipe for target 'fuzzer/sof-fuzzer' failed
make[2]: *** [fuzzer/sof-fuzzer] Error 1
CMakeFiles/Makefile2:85: recipe for target 'fuzzer/CMakeFiles/sof-fuzzer.dir/all' failed

Reply:
ahh figured it out. I had to change to PRIVATE from INTERFACE in target_link_libraries(sof-fuzzer PRIVATE ${GLIB_LDFLAGS})

Reply:
@ranj063 I didn't test code that I showed, it was just a tip ;p
But now I noticed, that it should be `PRIVATE` instead of `INTERFACE` in `target_link_libraries(sof-fuzzer INTERFACE ${GLIB_LDFLAGS})`. FYI INTERFACE just adds it to interface of sof-fuzzer (if it's going to be dep of something) but doesn't actually use it to link that target.

Reply:
@ranj063 Yea, I was minute too late with my answer

Reply:
@jajanusz makes sense. Thanks for your tips. I am not a cmake expert so I really appreciate your tips and comments.